values-schema.yaml

```diff
@@ -937,13 +1014,6 @@ properties:
                 $ref: '#/definitions/resources'
               pilot:
                 $ref: '#/definitions/resources'
-      jobs:
```

Comment:
These previous jobs need their own place. These jobs are hard named so maybe put them under "tasks". So it corresponds with `otomi-tasks`.

Reply:
Makes sense, will do.

Reply:
Well actually, just putting them back doesn't break anything. I don't know why I deleted them in the first place.

Reply:
That is fine indeed.